v2.1/learn-cockroachdb-sql.md

```diff
@@ -10,76 +10,6 @@ This page walks you through some of the most essential CockroachDB SQL statement
 Use an interactive SQL shell to try out these statements. If you have a cluster already running, use the [`cockroach sql`](use-the-built-in-sql-client.html) command. Otherwise, use the [`cockroach demo`](cockroach-demo.html) command to open a shell to a temporary, in-memory cluster.
 {{site.data.alerts.end}}
 
-{{site.data.alerts.callout_info}}
-CockroachDB aims to provide standard SQL with extensions, but some standard SQL functionality is not yet available. See our [SQL Feature Support](sql-feature-support.html) page for more details.
-{{site.data.alerts.end}}
-
-## Create a Database
```

Comment:
Why are we removing create database and show database? Even though the defaults have changed, these still seem like relevant actions to highlight here...